process/transaction/shardProcess.go

```diff
@@ -587,6 +569,8 @@ func (txProc *txProcessor) computeRelayedTxFees(tx *transaction.Transaction) (*b
 	totalFee := core.SafeMul(tx.GetGasLimit(), tx.GetGasPrice())
 	remainingFee := big.NewInt(0).Sub(totalFee, relayerFee)
 
+	//TODO: check tx.GasLimit - relayerGasLimit to be positive
```

Comment:
otherwise will fail the previous check of gasLimit.

Reply:
ok